climakitae/selectors.py

```diff
@@ -146,21 +190,36 @@ class LocSelectorArea(param.Parameterized):
         administrative geographic area]
     """
 
-    area_subset = param.ObjectSelector(
-        default="none",
-        objects=["none", "lat/lon", "states", "CA counties", "CA watersheds"],
+    area_subset = param.ObjectSelector(objects = dict())
+    cached_area = param.ObjectSelector(objects = dict())
+    default_lat = (32.5, 42)
+    default_lon = (-125.5, -114)
+    latitude = param.Range(
+        default = default_lat, 
+        bounds=(10, 67)
     )
-    # would be nice if these lat/lon sliders were greyed-out when lat/lon subset
-    # option is not selected
-    latitude = param.Range(default=(32.5, 42), bounds=(10, 67))
-    longitude = param.Range(default=(-125.5, -114), bounds=(-156.82317, -84.18701))
-    cached_area = param.ObjectSelector(objects=dict())
+    longitude = param.Range(
+        default = default_lon, 
+        bounds = (-156.82317, -84.18701)
+    )
+    _lat_lon_warning = param.String(
+        default = "", 
+        doc = "Warning if user is messing with lat/lon slider, \
```

Comment:
I think using a line cont. `\` will introduce extra spaces into your text string on the UI side. Maybe try using the ("","") method instead to split this up.